.github/workflows/ci.yml

```diff
@@ -114,6 +114,7 @@ jobs:
           py7zrversion: ==0.20.*
       - name: Install dependencies and build
         run: |
+          pip install nuitka minecraft_launcher_lib PySide6 click tomli darkdetect
```

Comment:
these should™ install from poetry since the reason these are included is the project uses them

Reply:
buut i forgot `poetry run` so this can work for now haha